pkg/analyzer/analyzers/launchdarkly/models.go

```diff
@@ -0,0 +1,135 @@
+package launchdarkly
+
+import "sync"
+
+var (
+	MetadataKey = "key"
+
+	// resource types
+	applicationKey  = "Application"
+	repositoryKey   = "Repository"
+	projectKey      = "Project"
+	environmentKey  = "Environment"
+	experimentKey   = "Experiment"
+	holdoutsKey     = "Holdout"
+	membersKey      = "Member"
+	destinationsKey = "Destination"
+	templatesKey    = "Templates"
+	teamsKey        = "Teams"
+	webhooksKey     = "Webhooks"
+	featureFlagsKey = "Feature Flags"
+)
+
+type SecretInfo struct {
+	User        User
+	Permissions []string
+	Resources   []Resource
+	// to concurrently read and write
+	mu sync.RWMutex
```

Comment:
nit: I've found it helpful to use `mu` almost like a **"hat"** for the resource(s) it protects by placing it directly above them. This makes it visually clear what it's guarding and can even replace the need for a comment.  

That said, I’m not entirely sure which resources `mu` is protecting in this case. 🤔 This is definitely an **opinionated** take, so feel free to ignore it! 🤣  

I forget where I first read about this, but [here](https://dmitri.shuralyov.com/idiomatic-go/entries/2) is a separate article about it. 😅 

Ex:
```
type SecretInfo struct {
    ...
    mu sync.RWMutex <-- see, like a "hat" for "Resources"
    Resources []Resource
}
```